Build error in `tools/payload.cc`

### Issue Type

Before opening an issue, please search and see if it has already been raised.

- [x] Bug Report
- [ ] Feature Request

- [x] Successfully reproduced against the latest version of NW.js?

### Current/Missing Behavior

When running `ninja build payload`, the following error is generated:
```
../../content/nw/tools/payload.cc:106:15: error: no matching member function for call to 'Append'
  file_list_->Append(base::WrapUnique(dict));
  ~~~~~~~~~~~~^~~~~~
../../base/values.h:690:8: note: candidate function not viable: no known conversion from 'std::unique_ptr<DictionaryValue>' to 'base::Value' for 1st argument
  void Append(Value&& value);
```

### Expected/Proposed Behavior

Like before 😁 

### Additional Info

- Operating System: macOS Monterey
- NW.js Version: 0.67.0
- Repro Link:
- Code snippet:
https://github.com/nwjs/nw.js/blob/37752063ed4f5e6a870dea0effba32d0089a8116/tools/payload.cc#L106
- Crash report:

--- a/tools/payload.cc
+++ b/tools/payload.cc
@@ -102,17 +102,17 @@ MyComputedHashes::Writer::~Writer() {
 void MyComputedHashes::Writer::AddHash(const base::FilePath& relative_path,
                                        int block_size,
                                        const std::string& root) {
-  base::DictionaryValue* dict = new base::DictionaryValue();
-  file_list_->Append(base::WrapUnique(dict));
-  dict->SetString(kPathKey,
-                  relative_path.NormalizePathSeparatorsTo('/').AsUTF8Unsafe());
-  //dict->SetInteger(kBlockSizeKey, block_size);
+  base::DictionaryValue dict;
+  dict.SetString(kPathKey,
+                 relative_path.NormalizePathSeparatorsTo('/').AsUTF8Unsafe());
+  //dict.SetInteger(kBlockSizeKey, block_size);
   std::string encoded;
   base::Base64Encode(root, &encoded);
   base::ReplaceChars(encoded, "=", "", &encoded);
   base::ReplaceChars(encoded, "/", "_", &encoded);
   base::ReplaceChars(encoded, "+", "-", &encoded);
-  dict->SetString(kBlockHashesKey, encoded);
+  dict.SetString(kBlockHashesKey, encoded);
+  file_list_->Append(std::move(dict));
 }
 
 bool MyComputedHashes::Writer::WriteToFile(const base::FilePath& path) {

